### PR TITLE
(FM-4966) Release unsupported 0.7.0

### PR DESCRIPTION
**Pull request by ferventcoder**
Update the Changelog, ReadMe, and metadata.json for release.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": ">= 3.0.0 < 2015.4.0"
+      "version_requirement": ">= 3.0.0 < 2016.2.0"
```
Comment (glennsarti) [inline metadata.json]: Can we change this to 2016.4.0 ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 2016-06-01 Unsupported Release 0.7.0
+
+ * Manage sources - `chocolateysource` - see [MODULES-3037](https://tickets.puppetlabs.com/browse/MODULES-3037)
+ * $::chocolateyversion fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
+ * Fix: puppet apply works again - see [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105)
+
+# Original Module
```
Comment (glennsarti) [inline CHANGELOG.md]: Will the term `Original` be confusing?  Perhaps `Original module published by Chocolatey` or something more descriptive?

File: README.md [hunk 3]
```diff
@@ -510,6 +559,46 @@ This is the **only** place in Puppet where backslash separators should be used.
...
+
+#### Properties/Parameters
+
+#### name
+(**Namevar**: If ommitted, this attribute's value will default to the resource's
+title.)
+
+The name of the source. Used for uniqueness. Will set the location to this value if location is unset.
+
+##### `ensure`
```
Comment (glennsarti) [inline README.md]: Seems like a typo.  Should be `#### ensure`

Comment (ferventcoder) [inline README.md]: I think the others are incorrect. Good catch

File: CHANGELOG.md (hunk 2 above)
Comment (ferventcoder) [inline CHANGELOG.md]: Published by Chocolatey team - yeah - will change.

File: metadata.json (hunk 1 above)
Comment (ferventcoder) [inline metadata.json]: Sure.

File: README.md (hunk 3 above)
Comment (ferventcoder) [inline README.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (ferventcoder) [inline CHANGELOG.md]: Done.

File: metadata.json (hunk 1 above)
Comment (ferventcoder) [inline metadata.json]: Done.

**Comment by ferventcoder** [comment]
@glennsarti @puppetlabs/windows this is ready

**Pull request merged**